src/main/resources/schemas/master_schema.json

```diff
@@ -2006,117 +868,43 @@
                                       },
                                       "offence": {
                                         "title": "offence",
-                                        "description": "The offence and details of offences associated with the case. It is also known as charges.",
                                         "type": "array",
                                         "items":{
                                           "type": "object",
                                           "properties": {
-                                            "offenceCode":{
```

Comment:
This is in Magistrates standard list and test files. If we remove from the master schema, these should be tidied up too in data-management, channel-management and frontend